examples/hot_reload_tide/src/main.rs

```diff
@@ -3,9 +3,7 @@
 // you can edit the configuration and the app will pick up changes without the need to restart it.
 // This concept is known as hot-reloading.
 use hot_reload_tide::messages::{load_config, Config};
-use notify::{
-    event::ModifyKind, Error, Event, EventKind, RecommendedWatcher, RecursiveMode, Watcher,
-};
+use notify::{Error, Event, RecommendedWatcher, RecursiveMode, Watcher};
```

Comment:
Is that done manually ? Because I think this was from rust fmt.

Reply:
No, I did this by `cargo fmt`. 
I think this is because the length of this line was changed less than the value of `max_width`(default value is 100).